rb/spec/integration/selenium/webdriver/element_spec.rb

```diff
@@ -94,6 +94,11 @@ module WebDriver
         expect(driver.find_element(id: 'withText').attribute('nonexistent')).to be_nil
       end
 
+      it 'should get property value' do
```

Comment:
Aha, I couldn't find that page, thanks!

Also, shouldn't this be

```
not_compliant_on browser: :edge do
```

Since we want to test the backwards compatibility as well?